queries/ini/highlights.scm

```diff
@@ -13,4 +13,4 @@
 (setting
   (setting_name) @property)
 
-; (setting_value) @none ; grammar does not support subtypes
+(setting_value) @string ; grammar does not support subtypes
```

Comment:
Is the comment still relevant?

Reply:
What if the `option = 1.0`? That should _not_ be captured as `@string`. I think the comment still applies and prevents this capture.

Reply:
But that is not guaranteed to be a number, a program may read that as whatever it wants since there is no spec to how to read an ini file. Also, for the [editorconfig queries](https://github.com/nvim-treesitter/nvim-treesitter/pull/6855#discussion_r1665476308) it was point out that default values should be highlighted as strings by default.

Reply:
Hmm, not sure I agree with that comment. The legacy syntax file also distinguishes them. If anything, it should be `@identifier` then.

Reply:
I don't see how that should be an `@identifier` and I don't see that capture in `:h treesitter-highlight-groups` or `CONTRIBUTING.md`

Reply:
Sorry, name is `@variable`.

Reply:
If the argument is that the value is _always_ treated as a string literal in these files, that's different, of course. I just disagree with the "default value" reasoning.